### PR TITLE
MCKIN-12116 for text provided for label of answer editable

### PR DESCRIPTION
**Pull request by musmanmalik**
There was no label for text provided for answer editable. while accessibility testing screen reader was not considering it a valid label while accessibility testing. Now  FOR/ID provided for that.

### CODE REVIEW

**Comment by musmanmalik** [comment]
@msaqib52  please review this

File: problem_builder/templates/html/answer_editable.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {% load i18n %}
 <div class="xblock-answer" data-completed="{{ self.completed }}">
   {% if not hide_header %}<h4 class="question-title">{{ self.display_name_with_default }}</h4>{% endif %}
-  <label><p>{{ self.question|safe }}</p>
+  <label for="label for answer textarea"><p>{{ self.question|safe }}</p>
```
Comment (nasirhjafri) [inline problem_builder/templates/html/answer_editable.html]: Let's add `-` instead of spaces in id.

**Comment by musmanmalik** [comment]
@xitij2000 Please review this pull request and create a release tag.

**Comment by xitij2000** [comment]
@musmanmalik Sure, I've created a ticket, and we should review it early next sprint.

**Comment by Agrendalath** [comment]
@musmanmalik, sorry, haven't noticed that the PR is ready. I'll review this tomorrow.

**Comment by musmanmalik** [comment]
@Agrendalath @xitij2000 any update redarding this?

**Comment by Agrendalath** [comment]
@musmanmalik, thank you for the contribution! I merged the PR and created the release.

**Pull request merged**